ui/src/criteria/attribute.tsx

```diff
@@ -46,29 +71,64 @@ class _ implements CriteriaPlugin<Data> {
   }
 
   generateFilter() {
-    if (this.data.selected.length === 0) {
-      return null;
-    }
+    let operands: tanagra.Filter[] | undefined;
```

Comment:
Please remove this variable, it is only used in more tightly scoped context, so there is no need to define it here. My preference would be that you just inline the operands directly in the return statements below:


```
return {
  operands: [...],
  operator: ...
};
```

If you really prefer a variable though, it should just be defined within the scope it is referenced, i.e. `const operands = [` at L75 below

Reply:
Thank you CH! That's an obsolete definition from version before. I improve this as you suggested.